src/main/java/com/hazelcast/kubernetes/HazelcastKubernetesDiscoveryStrategy.java

```diff
@@ -103,6 +104,35 @@ private String discoverZone() {
         return "unknown";
     }
 
+    /**
+     * Discovers the name of the node which the current Hazelcast member pod is running on.
+     * <p>
+     * Note: NODE_AWARE is available only for the Kubernetes API Mode.
+     */
+    private String discoverNodeName() {
+        if (DiscoveryMode.KUBERNETES_API.equals(config.getMode())) {
+            try {
+                String podName = System.getenv("POD_NAME");
```

Comment:
These lines look the same as in `discoverZone()`. Could you extract it to a method `String podName()`?